### PR TITLE
RO-1912 Remove site.yml

### PR DESCRIPTION
**Pull request by major**
This patch removes the `site.yml`, which was unmaintained and unused.

Issue: [RO-1912](https://rpc-openstack.atlassian.net/browse/RO-1912)

### CODE REVIEW

**Comment by odyssey4me** [comment]
recheck_cit_xenial_deploy_swift

**Comment by odyssey4me** [comment]
recheck_cit_xenial_deploy_swift

**Comment by odyssey4me** [comment]
I'm not sure why it's suddenly failing with this:

```
fatal: [ramxdsp-418-b2c1]: FAILED! => {"attempts": 6, "changed": true, "cmd": ["/openstack/venvs/maas-1.0.0/bin/python", "/opt/rpc-maas-tools/rpc-maas-tool.py", "--logfile", "/tmp/../../logs/rpc-maas-logs.txt", "verify-local"], "delta": "0:00:00.039702", "end": "2017-09-04 10:23:39.139080", "failed": true, "rc": 1, "start": "2017-09-04 10:23:39.099378", "stderr": "Traceback (most recent call last):\n  File \"/opt/rpc-maas-tools/rpc-maas-tool.py\", line 28, in <module>\n    import yaml\nImportError: No module named yaml", "stdout": "", "stdout_lines": [], "warnings": []}
```

File: rpcd/playbooks/setup-maas.yml [hunk 1]
```diff
@@ -20,4 +20,4 @@
     - ceph.ceph-common
 
 - include: maas-get.yml
-- include: /opt/rpc-maas/playbooks/site.yml
```
Comment (odyssey4me) [inline rpcd/playbooks/setup-maas.yml]: This needs to stay!

**Comment by major** [comment]
Ah, that's a different `site.yml` entirely.  I get it now. ;)

**Comment by evrardjp** [comment]
You could include logging in the other one, but that's not needed IMO.

**Pull request merged**